suite-common/wallet-utils/src/settingsUtils.ts

```diff
@@ -1,9 +1,19 @@
 import { PROTO } from '@trezor/connect';
+import { networksCompatibility as NETWORKS, NetworkSymbol } from '@suite-common/wallet-config';
 
 export const getLocalCurrency = (localCurrency: string) => ({
     value: localCurrency,
     label: localCurrency.toUpperCase(),
 });
 
-export const getAreSatoshisUsed = (bitcoinAmountUnit: PROTO.AmountUnit) =>
-    bitcoinAmountUnit === PROTO.AmountUnit.SATOSHI;
+export const getAreSatoshisUsed = (bitcoinAmountUnit: PROTO.AmountUnit, symbol: NetworkSymbol) => {
+    const areSatsDisplayed = bitcoinAmountUnit === PROTO.AmountUnit.SATOSHI;
+
+    const areUnitsSupportedByNetwork =
```

Comment:
consider to use `hasNetworkFeatures` util for it, this ignores settings for account types

Reply:
b35cfaa5c8581471bffc234228a1bffa4b1fc3fb